config/settings/production.py

```diff
@@ -139,7 +139,7 @@
     "SALESFORCE_ARTICLE_URL_PATH_PREFIX", default="/articles/Resource/"
 )
 SALESFORCE_ARTICLE_PREVIEW_URL_PATH_PREFIX = env(
-    'SALESFORCE_ARTICLE_URL_PATH_PREFIX',
+    'SALESFORCE_ARTICLE_PREVIEW_URL_PATH_PREFIX',
```

Comment:
Are there other places this change should be made until we clean up the settings?

Reply:
Yes. https://github.com/SFDO-Tooling/sfdoc/pull/235/commits/c6c0459b6bd1cce1645156dfc6d367d56480d637